### ISSUE TITLE
error: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by ShawnStewart** [denjucks/dex#5]
```
// dbConfig.ts

import Dex from "https://deno.land/x/dex@1.0.2/mod.ts";
// Also tried:
// import Dex from "https://deno.land/x/dex@1.0.2/mod-dyn.ts";
// import Dex from "https://raw.githubusercontent.com/denjucks/dex/master/mod.ts";

const dex = await Dex({
  client: "sqlite3",
  connection: {
    filename: "./database/database.db3",
  },
  useNullAsDefault: true,
});

export default dex;
```

When I run `deno cache dbConfig.ts`, download succeeds but compiling results in `error: No such file or directory (os error 2)`. Has anyone else ran into this problem?

**Comment by SquareMesh** [comment]
Yes, enountered same issue.  Deno version 1.0.5 but when downgraded to 1.0.0 the error disppeared.

**Comment by ShawnStewart** [comment]
@SquareMesh, I found that trying to export or initialize dex in 1.0.5 results in the original error.
```
export { default as Dex } from "https://deno.land/x/dex/mod.ts"
// or
import Dex from "https://deno.land/x/dex/mod.ts";
const dex = Dex({ client: "sqlite3" });
```

**Comment by la-colinares** [comment]
https://github.com/denoland/deno/issues/6057#issuecomment-639682850

Hello guys! We have the same issue with Dex and that's because the Dex is currently not yet supported the latest version of Deno. Kindly check this link to fix your problem.

**Comment by denjucks** [comment]
Try downgrading to deno v1.0.2 for now, as there appears to be an error in v1.0.5 on linux machines from what I understand. I'll keep this issue open for now, and update it as newer versions of deno are released.

**Comment by denjucks** [comment]
Should be fixed now with Deno v1.1.0 release. Gonna close this issue
